content/docs/guides/adopting/from_terraform.md

```diff
@@ -252,18 +330,70 @@ network_state = terraform.state.RemoteStateReference('network',
 {{% choosable language go %}}
 
 ```go
-// Terraform RemoteStateReference is not yet supported in Go.
-//
-// See https://github.com/pulumi/pulumi-terraform/issues/518.
+package main
+
+import (
+	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
+	"github.com/pulumi/pulumi/sdk/v2/go/pulumi/config"
+
+	"github.com/pulumi/pulumi-terraform/sdk/v2/go/state"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+
+        conf := config.New(ctx, "")
+        token := conf.RequireSecret("tfeToken")
+        organization := conf.Require("organization")
+        workspace := conf.Require("workspaceName")
+
+        state, err := state.NewRemoteStateReference(ctx, "remote-backend-state", &state.RemoteBackendStateArgs{
+            Organization: pulumi.String(organization),
+            Token:        pulumi.StringPtrInput(token.(pulumi.StringOutput)),
+            Workspaces: state.WorkspaceStateArgs{
+                Name: pulumi.String(workspace),
+            },
+        })
+        if err != nil {
+            return err
+        }
+
+        ctx.Export("test", state.Outputs)
```

Comment:
I don't see the equivalent of this in the other examples?

Reply:
gone!